<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemReader</class>
 <widget class="QWidget" name="TemReader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TemReader</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>511</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>serial</string>
    </attribute>
    <widget class="QPlainTextEdit" name="plainData">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>451</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btOpenStop">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>360</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Open/Stop</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>156</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comPort"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comBaudRate"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>BaudRate:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>DataBits:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Parify:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>StopBits</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Timeout:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>QueryMode:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comDataBits"/>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comParify"/>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comStopBits"/>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="comQueryMode"/>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="spinTimeout">
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btExit">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>420</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;exit</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbStatus">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>290</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>I am stand ready!</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>view</string>
    </attribute>
    <widget class="QComboBox" name="comStyle">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>style</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>186</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Temerature 1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="spinHiTemp"/>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="spinHumi"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Humidity</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="spinTemp"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Temperature 2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Temp Fahrenheits</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="spinTempf"/>
      </item>
     </layout>
    </widget>
    <widget class="QwtThermo" name="thermoHumi" native="true">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>60</width>
       <height>250</height>
      </rect>
     </property>
    </widget>
    <widget class="QwtDial" name="diaTemp" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>171</width>
       <height>171</height>
      </rect>
     </property>
     <property name="lineWidth" stdset="0">
      <number>4</number>
     </property>
    </widget>
    <widget class="QwtDial" name="diaHiTemp" native="true">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="lineWidth" stdset="0">
      <number>4</number>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtDial</class>
   <extends>QWidget</extends>
   <header>qwt_dial.h</header>
  </customwidget>
  <customwidget>
   <class>QwtThermo</class>
   <extends>QWidget</extends>
   <header>qwt_thermo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
